master/getting-started/openshift/installation.md

```diff
@@ -16,6 +16,11 @@ inventory file:
   - `openshift_use_calico=true`
   - `openshift_use_openshift_sdn=false`
 
+If you are using OCP v3.6.0 or a version of openshift-ansible that does not
+include [the calico_ipv4pool_cidr commit](https://github.com/openshift/openshift-ansible/pull/5111),
```

Comment:
Are users going to know if they have this commit? Is there a specific version we can say they need?

i.e. "<= v3.6.0"?